app/invocation/invocation_model.tsx

```diff
@@ -29,7 +29,7 @@ export default class InvocationModel {
   brokenTest: build_event_stream.BuildEvent[] = [];
   flakyTest: build_event_stream.BuildEvent[] = [];
   timeoutTest: build_event_stream.BuildEvent[] = [];
-  files: build_event_stream.NamedSetOfFiles[] = [];
+  files: Map<string, build_event_stream.IFile[]> = new Map();
```

Comment:
nit: consider making this `private` so people don't accidentally reference this instead of `getFiles`

Also consider adding a comment describing what the keys represent, or rename to something like `xToFilesMap`

Reply:
Done